fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java

```diff
@@ -2465,6 +2465,17 @@ private void createOlapOrLakeTable(Database db, CreateTableStmt stmt) throws Ddl
                 colocateTableIndex.removeTable(tableId, olapTable, false /* isReplay */);
             }
         }
+        if (olapTable.getTableProperty().getDynamicPartitionProperty().getEnable()) {
+            new Thread(() -> {
```

Comment:
does dynamic partition scheduler has execution thread pool? better to use the same thread pool and give it a name.

Reply:
dynamic partition scheduler is a background thread. it does not have a thread pool.